backends/guest/guest_svc_verify.c

```diff
@@ -71,9 +71,6 @@ static int parse_options(int key, char *arg, struct argp_state *state)
 		else if ((rc = validate_variables_arguments(args)) != SUCCESS)
 			return rc;
 		break;
-		argp_usage(state);
```

Comment:
would prefer:
```
		else if ((rc = validate_variables_arguments(args)) != SUCCESS)
                       argp_usage(state);
	         break;
```

Also reading you commit message, I think the host equivalent `case ARGP_KEY_SUCCESS` section makes sense. We have a series of else if's to check for issues. If there is an issue, then it doesn't break. Therefore it would flow into the argp_usae and return arg_parse_fail.